themes/digital.gov/layouts/partials/core/header.html

```diff
@@ -17,7 +17,7 @@
       </svg> Menu
     </button>
   </div>
-  <nav role="navigation" class="usa-nav">
+  <nav aria-label="Primary navigation" class="usa-nav">
```

Comment:
This adds a unique aria label, like the current USWDS header.

Reply:
Should we remove the "navigation" copy in the `aria-label`? It is duplicated on VoiceOver, but maybe needed for other devices?

<img width="1673" alt="Screen Shot 2023-02-13 at 2 00 03 PM" src="https://user-images.githubusercontent.com/104778659/218550225-46631059-18d4-4359-b5e4-5beac730ebee.png">